src/components/NestedInputCheckboxGroup.vue

```diff
@@ -0,0 +1,127 @@
+<template>
+  <section>
+    <div class="ft-nested-checkbox">
+      <label class="ft-nested-checkbox__label">
+        <input class="ft-nested-checkbox__input" type='checkbox' :checked="allChecked" @click="areNoneChecked">
+        <div class="ft-nested-checkbox__icon" :class="{someChecked: someChecked, allChecked: allChecked}">
+          <transition-group name="scale">
+            <AddIconCheckboxPartial 
+            v-if="!allChecked && someChecked" 
+            key="selected-partial"
+            class="ft-checkbox-selected"/>
+            <AddIconCheckboxSelected 
+            v-if="allChecked" 
+            key="selected"
+            class="ft-checkbox-selected"/>
+            <AddIconCheckboxUnselected 
+            v-if="noneChecked"
+            key="unselected"/>
+          </transition-group>
+        </div>
+        <div class="ft-nested-checkbox__label-content">
+          <span>
+            {{label}}
+          </span>
+        </div>
+      </label>
+      <div class="ft-nested-checkbox__wrap">
+        <slot></slot>
+      </div>
+    </div>
+  </section>
+</template>
+
+<script lang="ts">
+  export interface NestedChildren {
+    child:Vue
+  }
+
+  import {CheckboxGroup} from 'types'
+  import Vue from "vue"
+  //Consider this for recusively building a tree of components
+  //https://vuejsdevelopers.com/2017/10/23/vue-js-tree-menu-recursive-components/
+  import {
+    CheckboxSelected24 as AddIconCheckboxSelected,
+    CheckboxUnselected24 as AddIconCheckboxUnselected,
+    CheckboxPartial24 as AddIconCheckboxPartial
+    } from '@fishtank/icons-vue'
+import { Component } from 'vue-router/types/router';
```

Comment:
Looks like this was auto-imported at one point (and it's the incorrect `Component` anyway). Remove this line.